shader/clip_leaf.wgsl

```diff
@@ -187,8 +187,8 @@ fn main(
         // Fix up drawmonoid so path_ix of EndClip matches BeginClip
         let parent_clip = clip_inp[parent];
         let path_ix = parent_clip.path_ix;
-        let parent_ix = parent_clip.ix;
-        let ix = ~inp;
+        let parent_ix = min(parent_clip.ix, config.n_drawobj - 1u);
```

Comment:
I'm not a fan of this change, as if it triggers it will create UB and incorrect output.